content/shared/_channel_enumeration.textile

```diff
@@ -0,0 +1,22 @@
+The REST API now includes the endpoint:
+
+```[sh]
+  GET /channels
+```
+
+This enumerates all active channels in the application. This is a paginated API following the same API conventions as other paginated APIs in the "Ably REST library":/rest.
+
+The following parameters are supported:
+
+- limit := _100_ optionally specifies the maximum number of results to return. A limit greater than 1000 is unsupported<br>__Type: @integer@__
+- prefix := optionally limits the query to only those channels whose name starts with the given prefix<br>__Type: @string@__
+- by := _value_ optionally specifies whether to return just channel names (@by=id@) or "ChannelDetails":general/channel_lifecycle#channel-details (@by=value@)
+
+The credentials presented with the request must include the @list-channels@ permission for the wildcard resource ('@*@') or, if a @prefix@ is specified, for a resource that matches the @prefix@.
+
+Client libraries do not currently support this API, but it is usable via the generic "request API":/rest/usage#request.
```

Comment:
Client libraries do not currently provide a dedicated API to enumerate channels, but make this available using the "request" method.

Here is a trivial example:
...